browser/src/control/jsdialog/Widget.MenuButton.js

```diff
@@ -115,17 +115,7 @@ function _menubuttonControl (parentContainer, data, builder) {
 			}
 		};
 
-		var isSplitButton = data.applyCallback;
-
-		if (control.arrow) {
-			JSDialog.AddOnClick(control.button, clickFunction);
-		}
-		// make it possible to setup separate callbacks for split button
-		else if (isSplitButton) {
-			JSDialog.AddOnClick(control.button, data.applyCallback);
-		} else {
-			JSDialog.AddOnClick(control.button, clickFunction);
-		}
+		JSDialog.AddOnClick(control.button, clickFunction);
```

Comment:
SplitButton is one of the widgets in the weld:: library in LO - we have to support it!
It can appear in any dialog we get from core. Also color picker buttons have to be split buttons.
We had requirements wrt that, also it's standard in all the text editors.

Reply:
Please comment and clarify in the task description, I am just doing the task description requirements!.

Reply:
I mean you removed the feature by this change, please do not remove existing functionality.
Color picker button had 2 parts: `(apply last color) (open dropdown menu)`, after your change there is only single function: `(open dropdown menu)`. I wrote it in the previous revision of the PR already.
Requirement is just to not create regressions in this case.
Would be good to create cypress test for it (as it seems to be missing)